oe_theme.theme

```diff
@@ -1484,3 +1484,20 @@ function oe_theme_preprocess_paragraph__oe_banner(array &$variables): void {
 
   $cacheability->applyTo($variables);
 }
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function oe_theme_theme_suggestions_page_alter(array &$suggestions, array $variables) {
+  if ($node = \Drupal::routeMatch()->getParameter('node')) {
```

Comment:
Better checking if node is an actual node object, since we are gonna invoke a method on it below:

```php
  $node = \Drupal::routeMatch()->getParameter('node');
  if ($node instanceof \Drupal\node\NodeInterface) {
```

Also, will this work when asking for a revision of a node? Now sure if in that case the route has an element `node` containing the node object, we should manually check that.